cmd/user/Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM golang:1.13 AS buildenv
+# First stage: build the executable.
+FROM golang:1.14-alpine AS buildenv
 
 LABEL maintainer="Rafał Lorenz <vardius@gmail.com>"
 
+# Install git + SSL ca certificates.
+# Git is required for fetching the dependencies.
+# Ca-certificates is required to call HTTPS endpoints.
+RUN apk update && apk add --no-cache git ca-certificates tzdata && update-ca-certificates
```

Comment:
why do u need this ?

Reply:
Without it Google could not post the token to us returning an error like this:

> Post https://accounts.google.com/o/oauth2/token: x509: certificate signed by unknown authority

Links:

[x509-certificate-signed-by-unknown-authority-running-a-go-app-inside-a-docker-container](https://preslav.me/2019/08/11/x509-certificate-signed-by-unknown-authority-running-a-go-app-inside-a-docker-container/)

[smallest-secured-golang-docker-image](https://github.com/chemidy/smallest-secured-golang-docker-image/blob/master/Dockerfile)

[go-docker-https](https://brendanr.net/blog/go-docker-https/)